docs/application/native/guides/error/error-handling.md

```diff
@@ -39,7 +39,7 @@ How to query error codes for a certain API function is determined by how the fun
   BUNDLE_ERROR_OUT_OF_MEMORY /* Out of memory */
   ```
 
-To obtain the error message for a specific error code, use the `get_error_message()` function to query the meaning of each error code. The pointer returned is a static variable; you must not free it.
+To obtain the error message for a specific error code, use the `get_error_message()` function in [Common Error](../../api/common/latest/group__CAPI__COMMON__ERROR.html) to query the meaning of each error code. The pointer returned is a static variable; you must not free it.
```

Comment:
Hello @sgchoi5, everything seems to be in order now, except for the indent. That is why the text is not appearing on the same indent as the text above.

Please indent each line from Line 42 to 59 by two spaces or  "  ".

Reply:
added two spaces from line 42 to 59